NGitLab.Tests/Docker/GitLabTestContext.cs

```diff
@@ -254,6 +255,19 @@ public int GetRandomNumber()
             return RandomNumberGenerator.GetInt32(0, int.MaxValue);
         }
 
+        public void ReportTestAsInconclusiveIfVersionOutOfRange(SemanticVersion minVersion = null, SemanticVersion maxVersion = null)
```

Comment:
I think it would be better to use a package such as NuGet.Versioning and use a VersionRange.
- It is clearer if min/max are inclusive or exclusive
- It would simplify `13.10.99` as you can set something like `[,13.11)`

```suggestion
        public void ReportTestAsInconclusiveIfVersionOutOfRange(VersionRange range)
```